packages/hooks-react/src/useOffers.ts

```diff
@@ -37,10 +36,6 @@ const useOffers = () => {
     onSuccess: () => accountController.reloadSubscriptions({ delay: 7500 }), // @todo: Is there a better way to wait?
   });
 
-  useEffect(() => {
-    initialise();
-  }, [requestedMediaOffers, initialise]);
-
```

Comment:
@royschut this `useEffect` caused an extra re-render (unit tests found this), switching the `useMutation` to `useQuery` should have the same effect because the key depends on the `requestedMediaOffers` as well. 

What do you think?

Reply:
Sure, that should work, and simplifies as well. 👍